okonomiyaki/file_formats/_egg_info.py

```diff
@@ -142,6 +144,12 @@ def parse_rawspec(spec_string):
             res[key] = spec[key]
         except KeyError:
             raise InvalidMetadataField(key, InvalidMetadataField.undefined)
+
+    for k, v in res.items():
+        if isinstance(v, six.binary_type):
+            res[k] = decode_if_needed(v)
```

Comment:
Well I was thinking `{k: decode_if_needed(v) for k, v ....}` but if we're supporting 2.6 then nevermind :)